.github/CODEOWNERS

```diff
@@ -50,7 +50,7 @@
 /tools/perf-regression-finder/          @m-redding @jsquire
 
 # PRLabel: %azsdk-cli
-/tools/azsdk-cli/                       @benbp @scbedd @praveenkuttappan @smw-ms @Jeo02 @maririos @richardpark-msft @l0lawrence @timovv @samvaity
+/tools/azsdk-cli/                       @benbp @scbedd @praveenkuttappan @smw-ms @Jeo02 @maririos @richardpark-msft @l0lawrence @timovv @samvaity @haolingdong-msft @MaryGao @tadelesh @msyyc @haiyuazhang
```

Comment:
@haiyuazhang is missing **public** memberships to the `Microsoft` and `Azure` GitHub organizations.   See the [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) for more context.

You can verify the state of the account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.

Reply:
Thank so much @jsquire for the guidance! 
@haiyuazhang, could you please follow the guide to add yourself as public memberships to the `Microsoft` and `Azure` GitHub organizations? You can verify it by using the script from Jesse, or just rerun the CI, the CI would pass after you are done.

Reply:
<img width="824" height="322" alt="image" src="https://github.com/user-attachments/assets/19988ebd-b90b-4f72-9964-8a10ccb8d3fd" />